roadmap.md

```diff
@@ -40,6 +40,17 @@ the following listing, all deliverables are sorted by their priority.
    compliant. We also intend to participate actively in the development of the
    framework.
 
+   Enhancement: https://github.com/kubernetes/enhancements/issues/3027
+
+1. **Signing of release artifacts (Secure)**
+
+   https://github.com/kubernetes/release/issues/914
+
+   Outcome: Being able to GPG sign release artifacts, which also includes
+   container images.
```

Comment:
```suggestion
   Outcome: Being able to sign release artifacts, which also includes
   container images.
```

I've closed the [GPG signing issue](https://github.com/kubernetes/release/issues/914#issuecomment-1004557637) in favor of `cosign` (https://github.com/kubernetes/enhancements/issues/3031 and https://github.com/kubernetes/release/issues/2227).

Let's drop the GPG reference here.

Reply:
Fixed :+1: